src/GrumPHP/Task/Phpparser.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace GrumPHP\Task;
+
+use GrumPHP\Task\AbstractParserTask;
+use GrumPHP\Exception\RuntimeException;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+
+/**
+ * Php Parser task
+ */
+class Phpparser extends AbstractParserTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'php_parser';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = parent::getConfigurableOptions();
+        $resolver->setDefault('triggered_by', array('php'));
+
+        return $resolver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return ($context instanceof GitPreCommitContext || $context instanceof RunContext);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $config = $this->getConfiguration();
+
+        $files = $context->getFiles(false)->extensions($config['triggered_by']);
+        if (0 === count($files)) {
+            return;
+        }
+        $parseErrors = $this->parse($files);
+
+        if ($parseErrors->count()) {
+            throw new RuntimeException(sprintf(
+                "You have matched keywords in your commit:\n%s",
```

Comment:
I also would like to add some non-keywords related visitors in the futre, so maybe it's better to give another error mesage.